frontend/src/components/projects/ProjectsDetailedContent.js

```diff
@@ -47,6 +60,8 @@ const ProjectsDetailedContent = ({projects, projectsByID, isFetching, get}) => {
                 <Descriptions.Item label="Comments" span={4}>{project.comments}</Descriptions.Item>
             </Descriptions>
             <TrackingFieldsContent entity={project}/>
+            <Title level={4} style={{marginTop: '2rem'}}> Associated Samples </Title>
+            <ProjectsAssociatedSamples projectName={project.name} />
```

Comment:
Might be better to pass IDs around since all the entities are stored according to their ids.